cmd/apd.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"log"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/SkycoinProject/skycoin/src/cipher"
+	"github.com/SkycoinProject/skywire-peering-daemon/src"
+)
+
+func main() {
+	shutDownCh := make(chan os.Signal)
+	signal.Notify(shutDownCh, syscall.SIGTERM, syscall.SIGINT)
+
+	apd := src.NewApd()
+	pubKey, _ := cipher.GenerateKeyPair()
+
+	apd.PublicKey = pubKey.Hex()
```

Comment:
What do you think of passing public key to `Apd` constructor?

Reply:
I think the public key should  be generated in the `NewApd()` and passed to the `APD` defined struct